h2o-core/src/main/java/water/Scope.java

```diff
@@ -85,12 +85,11 @@ static private void track_impl(Scope scope, Key key) {
       scope._keys.peek().add(key);            // Track key
   }
 
-  static public void untrack( Key<Vec>[] keys ) {
+  static public void untrack(Iterable<Key<Vec>> keys) {
```

Comment:
Because this more convenient for the caller. There are really two alternatives here: either require `Key[]` and require all callers to convert their collections to primitive array; or use `Iterable<Key<Vec>>` and require a primitive collection to be wrapped into a collection type. Of these two the first alternative is worse because it loses generic type information on `Key`. The second alternative is also more versatile: you can pass in an `ArrayList`, a `HashSet`, a `Generator`, etc.